contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -128,7 +135,12 @@ def _transmit(self, envelopes):
                             _requests_map['retry'] = _requests_map.get('retry', 0) + 1  # noqa: E501
                         else:
                             _requests_map['exception'] = _requests_map.get('exception', 0) + 1  # noqa: E501
-
+                if self._is_stats_exporter() and \
+                    not state.get_statsbeat_shutdown() and \
+                        not state.get_statsbeat_initial_success():
+                    # If ingestion threshold during statsbeat initialization is reached, return back code to shut it down
+                    if _statsbeat_failed_to_ingest():
+                        return -2
```

Comment:
It would be nice to use some kind of constant instead of the value here, -2 is shutdown signal, but looking at the code here I have no clue what -1 means.

Reply:
-1 is the exception signal for telemetry in general. -2 is the shutdown signal for only statsbeat exporter.

Reply:
Well using these kinds of numbers instead of enumerators or constants is usually a pretty bad practice in other languages, code is harder to understand and maintain by other developers, maybe this is the way to go in Python, just my two cents here.

Reply:
is -2 introduce in this PR? if so, how much work does it take to refactor?